collect_app/src/main/java/org/odk/collect/android/utilities/ThemeUtils.java

```diff
@@ -27,18 +27,22 @@
 import org.odk.collect.android.preferences.GeneralSharedPreferences;
 import org.odk.collect.android.preferences.PreferenceKeys;
 
+import javax.inject.Inject;
+
 public final class ThemeUtils {
 
-    private final Context context;
-    private final GeneralSharedPreferences generalSharedPreferences;
+    @Inject
+    GeneralSharedPreferences generalSharedPreferences;
+    @Inject
+    Context context;
 
-    /**
-     * @param context Should only be an activity's context. This is because application context
-     *                is unaware of current activity's theme/attributes
-     */
-    public ThemeUtils(Context context) {
-        this.context = context;
-        generalSharedPreferences = new GeneralSharedPreferences(context);
+    @Inject
```

Comment:
Isn't putting an `@Inject` above constructor only used in the case of constructor injection?

Reply:
Here, it is being used to provide `ThemeUtils` as an injectable dependency instead of doing it in the `AppModule`